source/extensions/filters/listener/original_src/original_src_socket_option.h

```diff
@@ -13,6 +13,9 @@ namespace OriginalSrc {
  */
 class OriginalSrcSocketOption : public Network::Socket::Option {
 public:
+  /**
+   * Constructs a socket option which will set the socket to use source @c src_address
```

Comment:
"@c" ?

Reply:
Just formatting. Uses a fixed-width font: http://www.doxygen.nl/manual/commands.html#cmdc

Reply:
OK I see, I think we are using `` for that though I don't know if that's appropriate for Doxygen.

Reply:
yeah for consistency I prefer `.